src/components/ClaimForm.js

```diff
@@ -307,6 +308,7 @@ const mapStateToProps = (state, props) => ({
   claimAdmin: state.claim.claimAdmin,
   claimHealthFacility: state.claim.claimHealthFacility,
   generating: state.claim.generating,
+  isValid: state.claim.validationFields?.claimCode?.isValid,
```

Comment:
It is a good idea to precise here the name of validated field. It will help if we'll decide to implement more than one validated field in this component. This is what I mean:
` isClaimCodeValid: state.claim.validationFields?.claimCode?.isValid.` 
Remember to change the name in _canSave_ method.